### DISCUSSION TITLE
Welcome to LumiaWOA Discussions!

### DISCUSSION THREAD

**Discussion by gus33000** [WOA-Project/Community-Issues#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akas1955** [comment]
Hi,

Should we allow for Windows updates in Windows Settings after installing Windows 11 on Lumia 950 XL via WOA Deployer?
Thanks,

akas1955

**Reply by Simizfo** [reply]
Please open a separate discussion thread! And yes, you can allow updates.
